var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -100,15 +102,16 @@ class Axom(CachedCMakePackage, CudaPackage):
     depends_on("lua", when="+lua")
 
     depends_on("scr", when="+scr")
-    depends_on("kvtree@master", when="+scr")
+    depends_on("kvtree@main", when="+scr")
     depends_on("dtcmp", when="+scr")
 
     depends_on("raja~openmp", when="+raja~openmp")
     depends_on("raja+openmp", when="+raja+openmp")
     depends_on("raja+cuda", when="+raja+cuda")
 
     with when('+umpire'):
-        depends_on('umpire@5.0.1:5')
+        depends_on('umpire@6.0.0:', when='@0.6.0:')
+        depends_on('umpire@5:5.0.1', when='@:0.5.0')
```

Comment:
Looking over previous change and the issue that it was referencing, Axom pre 0.6.0 worked with Umpire 5.0.0 and 5.0.1.  I thinks this change is more accurate.  

It also adds the newer umpire for axom@0.6.0+